ansible/ci-build-builder.yml

```diff
@@ -1,16 +1,28 @@
 ---
 - name: Build and push collector image
-  hosts: all
+  hosts: "{{ build_hosts | default('all') }}"
 
   environment:
     BUILD_BUILDER_IMAGE: "true"
-    COLLECTOR_BUILDER_TAG: "{{ ansible_env.COLLECTOR_BUILDER_TAG }}"
-    PLATFORM: "{{ ansible_env.PLATFORM }}"
+    COLLECTOR_BUILDER_TAG: "{{ collector_builder_tag }}"
+    PLATFORM: "linux/{{ arch }}"
+
+  vars:
+    collector_root: "{{ ansible_env.HOME }}/collector"
 
   tasks:
+    - name: Clone repository
+      ansible.builtin.git:
+        repo: https://github.com/stackrox/collector
+        dest: "{{ collector_root }}"
+        version: "{{ collector_git_sha }}"
+        refspec: "+{{ collector_git_ref | replace('refs/', '') }}"
+        recursive: true
+      when: arch == "s390x"
```

Comment:
Not sure I follow, why to clone only for s390x?

Reply:
Because it's the only platform (currently) that is building on a separate machine to the GHA runner. All the other platforms can build from the existing checked out source